File: business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/util/MetricsUtils.java
package no.nav.sbl.dialogarena.soknadinnsending.business.util;

public class MetricsUtils {

    public static String navKontorTilInfluxNavn(String mottaker) {
        if (mottaker == null) {
            return "";
        }
        return mottaker
                .replace("NAV", "")
                .replace(",", "");
    }

    public static int getProsent(int partial, int total) {
        if (total == 0) return 0;
        double prosent = (double) partial/total * 100; // Cast til double for å få desimaler i delingen
        return (int) Math.round(prosent);
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/metrics/MetricsUtils.kt
package no.nav.sosialhjelp.soknad.metrics

import kotlin.math.roundToInt

object MetricsUtils {
    fun navKontorTilInfluxNavn(mottaker: String?): String {
        return mottaker?.replace("NAV", "")?.replace(",", "") ?: ""
    }

    fun getProsent(partial: Int, total: Int): Int {
        if (total == 0) return 0
        val prosent = partial.toDouble() / total * 100 // Cast til double for å få desimaler i delingen
        return prosent.roundToInt()
    }
}
